src/shipit_static_analysis/shipit_static_analysis/report/base.py

```diff
@@ -6,6 +6,7 @@
 import itertools
 from shipit_static_analysis.clang.tidy import ClangTidyIssue
 from shipit_static_analysis.clang.format import ClangFormatIssue
+from shipit_static_analysis.lint import MozLintIssue
 
 COMMENT_FAILURE_SHORT = '''
 C/C++ static analysis found {defects_tidy} in this patch{extras_comments}.
```

Comment:
Nit: This is no longer just `C/C++`. Please remove and just say `Static analysis found ...`.